modules/nw-dns-cache-tuning.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * networking/dns-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-dns-cache-tuning_{context}"]
+= Tuning the CoreDNS cache
+
+You can configure the maximum duration of both sucessful (positive) or unsuccessful (negative) caching done by CoreDNS. Tuning the duration of caching of DNS query responses can reduce the load for any upstream DNS resolvers.
+
+.Procedure
+
+. Edit the DNS Operator object named `default` by running the following command:
++
+[source,terminal]
+----
+$ oc edit dns.operator.openshift.io/default
+----
+
+. Modify the time-to-live (TTL) caching values:
++
+.Configuring DNS caching
+[source,yaml]
+----
+spec:
+  cache:
+    successTTL: 1h <1>
+    denialTTL: 0.5h10m <2>
+----
++
+<1> The string value `1h` is converted to its respective number of seconds by CoreDNS. If this field is left empty, the value is assumed to be `0s` and the cluster will use the internal default value of `900s` as a fallback.
+<2> The string value can be a combination of units such as `0.5h10m` and is converted to its respective number of seconds by CoreDNS. If this field is left empty, the value is assumed to be `0s` and the cluster will use the internal default value of `30s` as a fallback.
++
+[WARNING]
+====
+Setting TTL fields to low values could lead to an increased load on any upstream resolvers.
+====
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information on DNS cache tuning, see the link:https://coredns.io/plugins/cache/[CoreDNS cache documentation].
```

Comment:
This feature only allows for a very small subset of the caching parameters available in CoreDNS so I'm a little wary about adding this here since we don't support most of what's in that documentation. That said, it's fine to leave in if it's necessary but I'd prefer it be taken out. I will defer to your judgement @jldohmann :slightly_smiling_face:

Reply:
No problem, I can remove it 👍